build-system/server/app.js

```diff
@@ -241,7 +241,6 @@ app.get(
     '/examples/*.(min|max).html',
     '/test/manual/*.(min|max).html',
     '/test/fixtures/e2e/*/*.(min|max).html',
-    '/dist/cache-sw.(min|max).html',
```

Comment:
going to defer to @jridgewell on final LGTM since i don't know what happened to the cache sw project